core/controllers/admin_test.py

```diff
@@ -2438,6 +2438,27 @@ def test_update_username_creates_audit_model(self) -> None:
         self.assertEqual(
             username_change_audit_model.new_username, self.NEW_USERNAME)
 
+    def test_profile_picture_is_missing_raises_error(self) -> None:
```

Comment:
Please add more tests for this, or modify the existing ones to also check for the files.

Reply:
Added the part where we check if after updating the username we are able to access the image via new username. The check is present in the above test `test_update_username`.